### ISSUE TITLE
will change the deep learning framework from tensorflow to pytorch [The description is attatched below]

### ISSUE THREAD

**Issue by shravya312** [Devanik21/ISRO_Mining_Site_FINAL_APP#18]
A dynamic computation graph is a key feature of PyTorch that allows the framework to create a map of calculations while the program is running.

Key Points:
Built While Running: The graph is formed as you run your code, not before. This means it can change every time you execute it.

Flexible and Adaptive: You can easily change how your model behaves based on the data you provide. For example, if the input data is different, the graph adjusts accordingly.

Easy to Debug: Since the graph is created as the code runs, you can use regular Python tools to check for errors right where they happen, making it easier to fix problems.

**Comment by shravya312** [comment]
can u assign this task to me @Devanik21

**Comment by shravya312** [comment]
please assign me the task @Devanik21

**Comment by shravya312** [comment]
if u have any suggestions u can surely let me know @Devanik21

**Comment by Devanik21** [comment]
> A dynamic computation graph is a key feature of PyTorch that allows the framework to create a map of calculations while the program is running.
> 
> Key Points: Built While Running: The graph is formed as you run your code, not before. This means it can change every time you execute it.
> 
> Flexible and Adaptive: You can easily change how your model behaves based on the data you provide. For example, if the input data is different, the graph adjusts accordingly.
> 
> Easy to Debug: Since the graph is created as the code runs, you can use regular Python tools to check for errors right where they happen, making it easier to fix problems.


Alright, try!🫡

**Comment by shravya312** [comment]
Thanks a lot for giving me this opportunity @Devanik21 I will try my best!

**Comment by Devanik21** [comment]
> Thanks a lot for giving me this opportunity @Devanik21 I will try my best!

hello, are u working on it? @shravya312

**Comment by shravya312** [comment]
I am working on it

**Comment by Devanik21** [comment]
> I am working on it,its taking time I will submit pr as soon its done can u please add labels to pull request accepted the leader board is live the pr is not reflecting @Devanik21

I have updated the labels to official ones, now it will be counted.

**Comment by shravya312** [comment]
No not in the issue in the pr

**Comment by shravya312** [comment]
![image](https://github.com/user-attachments/assets/70e16903-b55d-4dfb-9953-beb444fc36b2)

**Comment by shravya312** [comment]
in the right side labels is none yet

**Comment by Devanik21** [comment]
Please wait

**Comment by shravya312** [comment]
haa sure

**Comment by shravya312** [comment]
the pr should be assigned to me and labels should be added
![image](https://github.com/user-attachments/assets/f45a1606-a237-48eb-97df-9d2d9a5c4684)

**Comment by Devanik21** [comment]
> the pr should be assigned to me and labels should be added ![image](https://private-user-images.githubusercontent.com/125795769/373932911-f45a1606-a237-48eb-97df-9d2d9a5c4684.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgyMDY3MjcsIm5iZiI6MTcyODIwNjQyNywicGF0aCI6Ii8xMjU3OTU3NjkvMzczOTMyOTExLWY0NWExNjA2LWEyMzctNDhlYi05N2RmLTlkMmQ5YTVjNDY4NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwNlQwOTIwMjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNDRhNjI3ZjRiMWQxOWY5NzAyZmVlYjJkMTZjZmRiMGZjOTA4N2M1MThlNjk4MWQ2ZjUzMWFlMzE3ZWQ3YzJhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.E80JBS5EI1VA8RUrU5WDxSskLNXpUf05zur-CsTu8Wc)

plz check if its updated

**Comment by shravya312** [comment]
Thanks a lot It is accecpted can u add hactoberfest-accepted label 😇 I am working on other issues I will pr as soon as it is done

**Comment by shravya312** [comment]
By today it will be done @Devanik21
